ui/src/components/media/remote_control.rs

```diff
@@ -193,14 +197,36 @@ pub fn RemoteControls(cx: Scope<Props>) -> Element {
                     // todo: send command
                 }
             },
-            Button {
-                icon: Icon::PhoneXMark,
-                appearance: Appearance::Danger,
-                text: cx.props.end_text.clone(),
-                onpress: move |_| {
-                    ch.send(CallDialogCmd::Hangup(call.id));
-                },
+
+            // TODO: This needs a a signal sent when you terminate a call before the other user joins
+            //   Once the signal is added, this part will either need to be updated to show a different button with different method
+            //   (how it is configured below), or changed to fire both events on click (no more if/else, a single button with two things happening on press)
+            if active_call_participants_joined.is_some() && active_call_participants_joined.unwrap() > 0 {
```

Comment:
Maybe unwrap_or_default() here?

```suggestion
            if active_call_participants_joined.is_some() && active_call_participants_joined.unwrap_or_default() > 0 {
```